CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ include (TestBigEndian)
 # SQL, put -D WITH_SQL=OFF on the command line.
 
 option (WITH_SQL "Build this project with SQL (libdbi) support" ON)
-option (WITH_AQBANKING "Build this project with aqbanking (online banking) support" ON)
+option (WITH_AQBANKING "Build this project with aqbanking (online banking and SWIFT file imporrt) support" ON)
```

Comment:
A small typo: imporrt -> import

Reply:
Thanks. Since the file import is about a lot more than SWIFT now I changed that too.